### PR TITLE
Updates to qwiklabs

### PR DESCRIPTION
**Pull request by cxhercules**
Some updates to qwiklabs Readme

### CODE REVIEW

File: README-QWIKLABS.md [hunk 1]
```diff
@@ -133,18 +140,93 @@ The infrastructure required by this project can be deployed by executing:
...
+
+Jump onto the Debian virtual machine, that has application running on host OS. In this environment there is no isolation, and portability is less efficient. In a sense the app running on the system has access to all the system and depending on other factors may not have automatic recovery of application if it fails. Scaling up this application may require to spin up more virtual machines and most likely will not be best use of resources.
+```
+gcloud compute ssh vm-webserver
+```
+
+List all processes:
+```
+ps -ef
+```
```
Comment (chrislovecnm) [inline README-QWIKLABS.md]: This using older syntax, please update to modern “ps aux” for example

File: README-QWIKLABS.md [hunk 2]
```diff
@@ -133,18 +140,93 @@ The infrastructure required by this project can be deployed by executing:
...
+user      3168  3167  0 07:00 pts/0    00:00:00 -bash
+user      3174  3168  0 07:00 pts/0    00:00:00 ps -ef
+apprunn+  7938     1  0 Mar19 ?        00:00:31 /usr/bin/python /usr/local/bin/gunicorn --bind 0.0.0.0:8080 prime-flask-server
+apprunn+ 21662  7938  0 Mar20 ?        00:00:01 /usr/bin/python /usr/local/bin/gunicorn --bind 0.0.0.0:8080 prime-flask-server
+```
+
+Jump onto the [Container-Optimized OS (COS)](https://cloud.google.com/container-optimized-os/). COS is an optimized operating system with small OS footprint, which is part of what makes it secure to run container workloads. It has cloud-init and has Docker runt time preinstalled. This system on its own could be great to run several containers that did not need to be run on a platform that provided higher levels of reliability.
+
+```
+gcloud compute ssh cos-vm
```
Comment (chrislovecnm) [inline README-QWIKLABS.md]: They might need region and zone. 

Also you should add the word “bash” without the quotes to your code blocks. It enables GitHub to choose better syntax highlighting.

File: README-QWIKLABS.md [hunk 3]
```diff
@@ -133,18 +140,93 @@ The infrastructure required by this project can be deployed by executing:
...
+```
+
+Docker list containers
+```
+docker ps
+```
+```
+CONTAINER ID        IMAGE                                                  COMMAND                  CREATED             STATUS              PORTS                    NAMES
+d147963ec3ca        gcr.io/cxh-migration-to-containers/prime-flask:1.0.2   "/usr/bin/dumb-init …"   39 hours ago        Up 39 hours         0.0.0.0:8080->8080/tcp   flaskservice
+```
```
Comment (chrislovecnm) [inline README-QWIKLABS.md]: We can have them do a docker exec as well and do a ps there.

**Comment by cxhercules** [comment]
Please test this

**Comment by cxhercules** [comment]
Please test this

**Comment by cxhercules** [comment]
Please test this

**Comment by cxhercules** [comment]
Please test this

**Comment by cxhercules** [comment]
Please test this

**Comment by emailbob** [comment]
test this please

File: README-QWIKLABS.md [hunk 4]
```diff
@@ -133,18 +140,93 @@ The infrastructure required by this project can be deployed by executing:
...
+PID   USER     TIME  COMMAND
+    1 apprunne  0:00 /usr/bin/dumb-init /usr/local/bin/gunicorn --bind 0.0.0.0:8080 prime-flask-server
+    6 apprunne  0:36 {gunicorn} /usr/local/bin/python /usr/local/bin/gunicorn --bind 0.0.0.0:8080 prime-flask-server
+    8 apprunne  1:20 {gunicorn} /usr/local/bin/python /usr/local/bin/gunicorn --bind 0.0.0.0:8080 prime-flask-server
+    9 apprunne  0:00 ps -ef
+```
+
+As you can see from last example, the our python application is now running in a container only running its process, and cannot see other processes. It is also running in a namespace which can further restrict it from other containers running on the system. 
+
```
Comment (emailbob) [inline README-QWIKLABS.md]: Trail space here which is causing CI to fail

**Comment by chrislovecnm** [comment]
@cxhercules you still have edits that I requested, once that is done reach out to another admin to get this merged.

**Comment by cxhercules** [comment]
Yes I made changes, and thought it weird they did not show, let me look at that.

**Comment by cxhercules** [comment]
@chrislovecnm the changes you requested are there but not sure why it is not reflected in github ui. https://github.com/GoogleCloudPlatform/gke-migration-to-containers/pull/18/commits/ae7d9771ea37c43639857f3c4c26faf07f37b43c

**Comment by geojaz** [comment]
test this please

**Comment by geojaz** [comment]
test this please

**Comment by cxhercules** [comment]
test this please

**Pull request merged**